File: Famoser.MassPass.Data/Entities/Communications/Response/Encrypted/EncryptedResponse.cs
using System;
using Famoser.MassPass.Data.Entities.Communications.Response.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Response.Encrypted
{
    public class EncryptedResponse : ApiResponse
    {
        public Uri DownloadUri { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Response/Encrypted/EncryptedResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 23.05.2016
 * Time: 09:48
 */

namespace Famoser\MassPass\Models\Response\Encrypted;


use Famoser\MassPass\Models\Response\Base\ApiResponse;

class EncryptedResponse extends ApiResponse
{
    public $DownloadUri;
}